protocol/messenger_communities.go

```diff
@@ -1354,17 +1367,18 @@ func (m *Messenger) AcceptRequestToJoinCommunity(request *requests.AcceptRequest
 		return nil, err
 	}
 
-	pk, err := common.HexToPubkey(requestToJoin.PublicKey)
-	if err != nil {
-		return nil, err
-	}
+	if community.IsControlNode() {
```

Comment:
This is needed, because otherwise, the user received the acceptance/declining from admins.